libsolidity/analysis/OverrideChecker.cpp

```diff
@@ -573,6 +573,16 @@ void OverrideChecker::checkOverride(OverrideProxy const& _overriding, OverridePr
 			);
 	}
 
+	if (_overriding.unimplemented() && !_super.unimplemented())
+		overrideError(
+			_overriding,
+			_super,
+			4593_error,
+			"Overriding an implemented " + _super.astNodeName() +
```

Comment:
Can this lead to `variable declaration` here?

Reply:
I don't think so unless there's a way to create an unimplemented getter.

I'd add an assert just in case:
```c++
solAssert(!_overriding.isVariable() || !_overriding.unimplemented(), "");
```